test/document.spec.js

```diff
@@ -546,22 +547,52 @@ describe('Document', () => {
 
   describe('Lock Status', () => {
     describe('#lock', () => {
-      it('should lock the document', () => (
+      const lockDocument = () => {
```

Comment:
Please use `(` instead of `{` here, to make sure the result is returned, the `Promise` object, so that if the Promise rejects it will fail the test.